wp-content/themes/core/assets/css/src/theme/typography/_mixins.pcss

```diff
@@ -210,3 +210,17 @@
 		}
 	}
 }
+
+/* -------------------------------------------------------------------------
+ * Tag
+ * ------------------------------------------------------------------------- */
+
+@define-mixin t-tag {
+	display: inline-block;
```

Comment:
super minor, but there's a white space issue here. A quick `gulp lint` in the theme directory should solve this and any other little things.